.github/workflows/build-package.yml.template

```diff
@@ -109,16 +112,30 @@ jobs:
       - name: Tag release
         run: |
           (
+            set -euxo pipefail
             cd trigger
-            git tag $CRATE-$VERSION-$TARGET_ARCH
-            git push origin $CRATE-$VERSION-$TARGET_ARCH
+            git tag -f $CRATE-$VERSION $CRATE-$VERSION-$TARGET_ARCH
+            git push origin --tags
           )
 
-      - name: Releasing assets
-        uses: softprops/action-gh-release@v1
+      - name: Releasing assets in new release format
+        uses: svenstaro/upload-release-action@v2
         with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag_name: $CRATE-$VERSION
+          release_name: $CRATE-$VERSION
+          body: build $CRATE@$VERSION
+          file: |
+            ${{github.workspace}}/built.d/$CRATE-$VERSION-$TARGET_ARCH.tar.gz
+          overwrite: true
+
+      - name: Releasing assets in old release format
```

Comment:
Our checking code only checks the new location, so if this step fails we will never retry it. Maybe we should switch the order that we upload things so that this one comes first?

Reply:
I've switched the order so that it can be retried on failure.